source/macos/AdaptiveCards/AdaptiveCards/Renderers/BaseCardElementRenderer.swift

```diff
@@ -42,9 +37,24 @@ class BaseCardElementRenderer {
         }
         
         view.identifier = .init(element.getId() ?? "")
-        updatedView.addArrangedSubview(view)
         updatedView.isHidden = !element.getIsVisible()
         
+        guard let inputElement = element as? ACSBaseInputElement else {
+            updatedView.addArrangedSubview(view)
+            return updatedView
+        }
+        if let label = inputElement.getLabel(), !label.isEmpty {
```

Comment:
Guess this condition can be clubbed?
```suggestion
        if let inputElement = element as? ACSBaseInputElement, let label = inputElement.getLabel(), !label.isEmpty {
```

Reply:
Just add a comment like `Input label handling` in case this can be clubbed.